### PR TITLE
Polish deploy-multi-chain script

### PR DESCRIPTION
**Pull request by smol-ninja**
Closes https://github.com/sablier-labs/v2-core/issues/743

### CODE REVIEW

**Comment by smol-ninja** [comment]
@andreivladbrg let me know if looks good.

**Comment by andreivladbrg** [comment]
>I had to change MAINNET_RPC_URL with RPC_URL_MAINNET in foundry.toml because fork test was failing as we use RPC_URL_MAINNET in github secret.

github secrets can be updated as well, my intention is to switch from RPC_URL_{CHAIN_NAME} to {CHAIN_NAME}_RPC_URL because it is easier to group them and to alphabetically order in `.env`.

**Comment by smol-ninja** [comment]
Agree. I've changed it back https://github.com/sablier-labs/v2-core/pull/744/commits/083d6d7b4b395fa2f40920fbf59d1cd8b6d6bc06

File: shell/deploy-multi-chains.sh [hunk 1]
```diff
@@ -39,15 +51,26 @@ ARBITRUM_CHAIN_ID="42161"
...
+if [ -f .env ]; then
+    source .env
+else
+    echo -e "${EC}Error: .env file not found${NC}"
+    exit 1
+fi
+
+# Check: required Bash >=4.0.0 for associative arrays
+if ((BASH_VERSINFO[0] < 4)); then
+    echo -e "${EC}Error:\nThis script requires Bash version 4.0.0 or higher.\nYou are currently using Bash version ${BASH_VERSINFO[0]}.${BASH_VERSINFO[1]}.${BASH_VERSINFO[2]}.\nPlease upgrade your Bash version and try again.${NC}"
```
Comment (andreivladbrg) [inline shell/deploy-multi-chains.sh:71]: readability can be increased

**Review by andreivladbrg — CHANGES_REQUESTED** [review]
I've added the comptrollers here with the intention of adding a deploy core 3 script that does not deploy the comptroller. Can you also verify if there have been any changes between `release` and `staging` in `src/SablierV2Comptroller` or `src/interfaces/ISablierV2Comptroller`?

**Comment by andreivladbrg** [comment]
pushed two commits, lmk if it looks good now

**Comment by smol-ninja** [comment]
LGTM!

**Pull request merged**